docs/user-guide/rug/conventions.md

```diff
@@ -101,288 +70,262 @@ the [`CHANGELOG.md` format][changelog].
 
 [changelog]: http://keepachangelog.com/
 
-### Rug DSL and TypeScript Files
+### Project Naming
 
-Rug files should have the `.rug` extension when written using the
-DSL and the standard TypeScript `.ts` extension when written in
-TypeScript.  You can safely intermix Rug DSL and TypeScript Rugs in
-the same project.  Rug files should placed in the following locations
-within a _Rug project_.  Rug editors and generators should be in
-`.atomist/editors` and Rug reviewers should be in `.atomist/reviewers`.
+A Rug project name should be hyphenated and start with the technology
+stack being targeted, such as `spring-boot`, followed by `-editors` if
+this is a Rug project with a collection of useful editors or reviewers.
 
-BDD-style tests for your Rugs are strongly recommended and should be
-located within a `.atomist/tests` directory and have the `.rt`
-extension.
+If the main purpose of a Rug project is to be a generator, then ending
+the name of the type of project it will generate would be most
+appropriate.  For example if your Rug projects's main purpose was to be
+a generator for a Spring Boot Rest Service then
+`spring-boot-rest-service` or even `java-spring-boot-rest-service`
+would be appropriate. If your Rug projects's main purpose was simply to
+generate a valid Maven project then `maven-project` would be appropriate.
 
-If your Rugs, typically editors, use any templates, they are placed in
-the `.atomist/templates` directory.
+### Configuration File
```

Comment:
I prefer "Metadata" to "Configuration", since it is not really configuring anything.